app/views/posts/show.html.erb

```diff
@@ -7,10 +7,22 @@
 
         <div class="comments-post">
             <p class="card-text"><strong>Comments:</strong> <%= @post.comments_counter %></p>
+            <%= form_for [@post, Like.new], url: post_likes_path(@post) do |f| %>
+                <%= f.hidden_field :user_id, value: current_user.id %>
+                <%= f.submit "Like", class: "like-button" %>
+            <% end %>
             <p class="card-text"><strong>Likes:</strong> <%= @post.likes_counter %></p>
         </div>
     </div>
-  
+
+    <%= form_for [@post, Comment.new] do |f| %>
+        <div class="field">
+             <%= f.label :text %>
+          <%= f.text_area :text %>
+        </div>
+
+         <%= f.submit "Add Comment" %>
+    <% end %>
```

Comment:
- [ ] Great progress on the form implementation! 👏 Your effort is evident. I'd like to suggest a slight improvement in the structure of your routes, specifically for the **comment form**.

   - Consider creating a dedicated route for the comment form, such as `/posts/:post_id/comments/new`. Currently, you're rendering the form on `/users/:user_id/posts/:id`, which is intended for viewing post details. Separating the comment form onto its own route enhances the organization and clarity of your application.

   - After successfully submitting a comment, it's good practice to redirect users to the post details page (`/users/:user_id/posts/:id`). This provides a seamless user experience, allowing them to  see their comment in the context of the post. See below for more details on your UI:
<img width="1072" alt="Screenshot 2023-11-10 at 10 45 58" src="https://github.com/iankiema/BlogApp/assets/107192076/c5b3eea4-b724-455b-9a69-d28ae8644a36">

   

Making these adjustments ensures a cleaner structure and a more intuitive user flow. 
Happy doing! 😊🚀